mvc/migration/migratingconfig.rst

```diff
@@ -1,81 +1,55 @@
+  :language: c#
+  :linenos:
+  :lines: 15-24
+  :dedent: 8
+  
+Note that at this point the *Startup.cs* file will not compile, as we still need to add the following ``using`` statement:
 
 .. code-block:: c#
 
-	public IConfiguration Configuration { get; set; }
+	using Microsoft.Extensions.Configuration;
 
-	public Startup(IHostingEnvironment env, IApplicationEnvironment appEnv)
-	{
-		var configurationBuilder = new ConfigurationBuilder()
-			.SetBasePath(appEnv.ApplicationBasePath)
-			.AddJsonFile("config.json");
-		Configuration = configurationBuilder.Build();
-	}	
+Finally, add an *appsettings.json* file to the root of the project using the appropriate item template:
 
-Note that at this point the Startup.cs file will not compile, as we still need to add some using statements and pull in some dependencies. Add the following two using statements:
+.. image:: migratingconfig/_static/add-appsettings-json.png
+    :width: 955px
 
-.. code-block:: c#
-
-	using Microsoft.Framework.Configuration;
-
-Next, open project.json and add the Microsoft.Framework.ConfigurationModel.Json dependency:
-
-.. code-block:: javascript
-
-	{
-		"webroot": "wwwroot",
-		"version": "1.0.0-*",
-		"dependencies": {
-			...
-			"Microsoft.Framework.Configuration": "1.0.0-beta8",
-			"Microsoft.Framework.Configuration.Json": "1.0.0-beta8"
-		},
-		...
-	}
-
-Finally, add a config.json file to the root of the project.
-
-.. image:: migratingconfig/_static/add-config-json.png
-
-Migrate Configuration Settings from Web.config
+Migrate Configuration Settings from web.config
 ----------------------------------------------
 
-Our ASP.NET MVC 5 project included the required database connection string in Web.config, in the <connectionStrings> element. In our MVC 6 project, we are going to store this information in the config.json file. Open Config.json, and you should see that it already includes the following:
+Our ASP.NET MVC 5 project included the required database connection string in *web.config*, in the ``<connectionStrings>`` element. In our MVC 6 project, we are going to store this information in the *appsettings.json* file. Open *appsettings.json*, and note that it already includes the following:
 
-.. code-block:: javascript
+.. literalinclude:: migratingconfig/samples/WebApp1/src/WebApp1/appsettings.json
+  :language: json
+  :emphasize-lines: 4
+  :linenos:
 
-	{
-		"Data": {
-			"DefaultConnection": { 
-				"ConnectionString": "Server=(localdb)\\MSSQLLocalDB;Database=_CHANGE_ME;Trusted_Connection=True;"
-			}
-		}
-	}
-
-Change the name of the Database from _CHANGE_ME. In the case of this migration, we are going to point to a new database, which we'll name NewMvc6Project to match our migrated project name.
+In the highlighted line depicted above, change the name of the database from **_CHANGE_ME**. In the case of this migration, we are going to point to a new database, which we'll name *NewMvc6Project* to match our migrated project name.
```

Comment:
@Rick-Anderson How does the following sound?

In the highlighted line depicted above, change the name of the database from **_CHANGE_ME**. We are going to point to a new database, which will be named *NewMvc6Project* to match our migrated project name.